src/include/common/container/concurrent_vector.h

```diff
@@ -85,6 +85,18 @@ class ConcurrentVector {
    */
   Iterator PushBack(const T &item) { return Iterator(vector_.push_back(item)); }
 
+  /**
+   *
+   * Adds a new element at the end of the vector, after its current last element.
+   * @tparam Args template for arguments to element's constructor
+   * @param argsm arguments to construct element to be added
+   * @return an iterator pointing to the new element.
+   */
+  template <typename... Args>
+  Iterator EmplaceBack(Args &&... argsm) {
+    return Iterator(vector_.emplace_back(argsm...));  // NOLINT
```

Comment:
The NOLINT doesn't appear to be doing anything so remove it (it's still failing to find the backend function in clang-tidy in Travis).